doc/source/ray-core/ray-dashboard.rst

```diff
@@ -82,57 +314,96 @@ The information of up to 1000 dead actors will be stored.
-Instructions about that can be found :ref:`here <ray-metrics>`.
+From the actor detail page, you can see the metadata, state, and the all tasks that have run from this actor. 
 
-You can select the time range of the metrics in the top right corner. The graphs refresh automatically every 15 seconds.
+.. _dash-metrics-view:
 
-There is also a convenient button to open the grafana UI from the dashboard. The Grafana UI provides additional customizability of the charts.
+Metrics View
+------------
 
 .. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard/metrics.png
     :align: center
 
-Event View
-~~~~~~~~~~
+Ray exports default metrics which are available from the :ref:`Metrics View <dash-metrics-view>`. Here are some available example metrics.
+
+- The tasks, actors, and placement groups broken down by states.
+- The :ref:`logical resource usage <logical-resources>` across nodes.
+- The hardware resource usage across nodes.
+- The autoscaler status.
+
+See :ref:`System Metrics Page <system-metrics>` for available metrics.
 
 .. note:: 
 
-  The event view feature is experimental.
+  The metrics view required the Prometheus and Grafana setup. **See :ref:`Ray Metrics <ray-metrics>` to learn how to set up Prometheus and Grafana**.
 
-The event view lets you see a list of events associated with a specific type (e.g., autoscaler or job) in a chronological order. The equivalent information is also accessible via CLI commands ``ray list cluster-events`` :ref:`(Ray state APIs)<state-api-overview-ref>`.
+The metrics view lets you view visualizations of the time series metrics emitted by Ray.
 
-There are 2 types of events that are available.
+You can select the time range of the metrics in the top right corner. The graphs refresh automatically every 15 seconds.
 
-- **Job**: Events related to :ref:`Ray job submission APIs <jobs-quickstart>`.
-- **Autoscaler**: Events related to :ref:`Ray autoscaler <cluster-autoscaler>`
+There is also a convenient button to open the grafana UI from the dashboard. The Grafana UI provides additional customizability of the charts.
+
+.. _dash-logs-view:
 
-.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard/event.png
+Logs view
+---------
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard/logs.png
     :align: center
 
+The logs view lets you view all the ray logs that are in your cluster. It is organized by node and log file name. Many log links in the other pages will link to this view and filter the list so the relevant logs appear.
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard/logs-content.png
+    :align: center
+
+The log viewer provides various search functionality to help find the log messages you are looking for.
+
 Advanced Usage
 --------------
 
+Change the dashboard ports
```

Comment:
The names of the tabs might lead to confusion. The options are not mutually exclusive.
What if people use "CLI" and "VM launcher" together?

To me, it should be:
- SIngle-node local cluster
- VM cluster launcher
- KubeRay

Reply:
Hmm this seems fine. The VM cluster launcher explanation already includes the sufficient information.

Reply:
No. It doesn't really make sense. For example, users might try the following instruction under the `python` tab when they use VM cluster launcher or kuberay. Neither of it will work.
> you can pass the keyword argument dashboard_port in your call to ray.init()

Reply:
So, your suggestion is to use SIngle-node local cluster and write both ray.init() & ray start examples in there right? that sounds reasonable.

Reply:
Yep. Basically, these are the two ways of creating a single-node local cluster